govtool/frontend/src/i18n/locales/en.ts

```diff
@@ -275,7 +275,7 @@ export const en = {
         "Select this to signal no confidence in the current constitutional committee by voting NO on every proposal and voting YES to no confidence proposals",
       noConfidenceDefaultTitle: "Signal No Confidence on Every Vote",
       noResultsForTheSearchTitle: "No DReps found",
-      noResultsForTheSearchDescription: "Please try a different search",
+      noResultsForTheSearchDescription: "If you can't find the DRep you're looking for, either they are not registered, requested not to be listed in the DRep Directory, or did not submit metadata during registration.",
```

Comment:
Technically another case is when metadata is submitted but not correct, so maybe `...or did not submit correct metadata during registration.`

Reply:
I agree with your suggestion @jdyczka

Reply:
```suggestion
      noResultsForTheSearchDescription: "If you can't find the DRep you're looking for, either they are not registered, requested not to be listed in the DRep Directory, or did not submit correct metadata during registration.",
```